src/main/frontend/src/components/WordShow.js

```diff
@@ -0,0 +1,31 @@
+import React, {useState, useEffect } from 'react'
+import ReviewForm from './ReviewForm'
+
+const WordShow = (props) => {
+  let wordId = props.match.params.id
+  const [pet, setPet] = useState([])
+  const [newReview, setNewReview] = useState([])
+
+  useEffect(() => {
+   fetch('api/v1/${props.location.pathname}')
+    .then(result => result.json())
+    .then(word => {
+      setWord(word)
```

Comment:
Here I see you use setWord, but I do not know where this function is on the page?

Reply:
This will need to be changed in order for it to work properly.